qemu/tests/cfg/hotplug_virtio_mem.cfg

```diff
@@ -0,0 +1,37 @@
+- hotplug_virtio_mem:
+    only Linux
+    only Host_RHEL.m9
+    only RHEL.9
+    no s390x
+    type = hotplug_virtio_mem
+    virt_test_type = qemu
+    login_timeout = 240
+    threshold = 0.025
+    smp = 8
```

Comment:
maybe we can move this line  'smp = 8 ' under aarch64: directly.

Reply:
Changed now, thanks!